lmdeploy/serve/proxy/proxy.py

```diff
@@ -263,7 +306,12 @@ def handle_api_timeout(self, node_url):
         }
         return json.dumps(ret).encode() + b'\n'
 
-    async def stream_generate(self, request: Dict, node_url: str, endpoint: str):
+    async def stream_generate(self,
+                              request: Dict,
+                              node_url: str,
+                              endpoint: str,
+                              prefill_url: Optional[str] = None,
```

Comment:
Is `prefill_url` necessary? We can tell what the role of the node is by given `node_url`, cann't we?